api/models/community.js

```diff
@@ -74,6 +75,76 @@ export const getCommunitiesByUser = (userId: string): Promise<Array<DBCommunity>
   );
 };
 
+// prettier-ignore
+export const getVisibleCommunitiesByUser = async (evaluatingUserId: string, currentUserId: string) => {
+  const evaluatingUserMemberships = await db
+    .table('usersCommunities')
+    // get all the user's communities
+    .getAll(evaluatingUserId, { index: 'userId' })
+    // only return communities the user is a member of
+    .filter({ isMember: true })
+    // get the community objects for each community
+    .eqJoin('communityId', db.table('communities'))
+    // get rid of unnecessary info from the usersCommunities object on the left
+    .without({ left: ['id', 'communityId', 'userId', 'createdAt'] })
+    // zip the tables
+    .zip()
```

Comment:
`zip() ` 🤔
is ti RethinkDB term😅❓

Reply:
Yes, this is the RethinkDB querying API

Reply:
@brianlovin wow, thank you so much😄and early morning!🍞☕️🌄